src/features/DisplayControls/NursingTasks/components/UpdateNursingTasks.jsx

```diff
@@ -403,6 +404,29 @@ const UpdateNursingTasks = (props) => {
     setInvalidText(invalidTimeText);
   };
 
+  const disableDoneTogglePostNextTaskTime = (medicationTask) => {
+    const filteredSlots = groupSlotsByOrderId[medicationTask.orderId];
+    const taskWithJustGreaterTime = [];
+    for (let i = 0; i < filteredSlots.length; i++) {
```

Comment:
Maybe we can do this instead of the for loop if we are just finding the next task:
` const taskWithJustGreaterTime = filteredSlots.find(
  slot => medicationTask.startTimeInEpochSeconds < slot.startTimeInEpochSeconds
);`